test/common.js

```diff
@@ -47,6 +47,7 @@ const config = {
   username: 'valid@email.com',
   apiKey: 'randomcharactersonlyq32YIpC8aMSH',
   apiURL: 'valid.realm.url/api/v1',
+  realm: 'valid.realm.url/api',
```

Comment:
Again, this is me just being my innocent doesn't-have-a-clue-about-JavaScript self, but why do we need to explicitly set the realm here?

Reply:
This is because while testing the index.js file, the module first goes through the code where it processes the config. The other tests worked fine with just the final versions of information (apiURL), but here, the module tries to generate the apiURL using the realm that the user would pass.